components/contact.module.css

```diff
@@ -0,0 +1,108 @@
+.contact_section_container {
+	width: 100%;
+	height: 100%;
+	display: grid;
+	grid-template-columns: 100%;
+	grid-template-rows: 15% 10% 75%;
+	justify-content: center;
+	align-items: center;
+	padding: 0 3rem;
+}
+
+.thank_you {
+	text-align: center;
+	margin: 0;
+}
+
+.form {
+	display: flex;
+	/* position: relative; */
+	flex-direction: column;
+	align-items: center;
+	justify-content: space-evenly;
+	width: 60%;
+	max-width: 400px;
+	height: 100%;
+	margin: 3rem auto;
+	font-family: var(--text-font);
+}
+
+.form input,
+.form textarea,
+.form button {
+	appearance: none;
+	outline: none;
+}
+
+.form input,
+.form textarea {
+	font-size: 1.5rem;
+	font-family: var(--text-font);
+	border: none;
+	border-bottom: 1px solid gray;
+	width: 100%;
+	height: 5rem;
+	position: relative;
+	transition: all 0.5s ease-out;
+}
+
+.form input::placeholder,
+.form textarea::placeholder {
+	font-weight: 600;
+	position: absolute;
+	top: 50%;
+	transform: translateY(-50%);
+}
+
+.form textarea {
+	resize: vertical;
+	max-height: 20rem;
+	padding: 1.5rem 0;
+}
+
+.form input:focus,
+.form textarea:focus {
+	border-bottom: 1px solid var(--accent-color);
+	box-shadow: 0 0.5rem 0.5rem -0.5rem var(--accent-color);
+}
+
+.form textarea:focus {
+	height: 20rem;
+}
+
+.form button {
+	font-family: var(--text-font);
+	font-weight: 600;
+	letter-spacing: 1px;
+	border: none;
+	background-color: var(--accent-color);
+	display: inline-block;
+	padding: 1rem 3rem;
+	box-shadow: 0.5rem 0.75rem 1rem 0 gray;
```

Comment:
Good job with the shadow! I'll have to apply something similar to audio-player-section's images